CITATION.cff

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Johnson"
+  given-names: "Alex"
+  orcid: https://orcid.org/0000-0003-4623-4147
+- family-names: "Tétreault-Pinard"
+  given-names: "Étienne"
+- family-names: "Samimi"
+  given-names: "Mojtaba"
+title: "Open-source charting library behind Plotly"
```

Comment:
"Open source Plotly charting library" (I don't think "behind" is quite right since it can be used on its own.)